CHANGELOG.md

```diff
@@ -23,7 +23,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `AmazonS3FileManagerDriver` for managing files on Amazon S3.
 - `MediaArtifact` as a base class for `ImageArtifact` and future media Artifacts.
 - Optional `exception` field to `ErrorArtifact`.
-- `GriptapeCloudStructureRunClient` tool for invoking Griptape Cloud Structure Run APIs.
+- `StructureRunClient` for running other Structures via a Tool.
```

Comment:
ugh why can't it end with Tool

Reply:
Named it this way for consistency `TaskMemoryClient`, another Tool that interacts directly with the framework. Agree that it's not ideal but it's at least consistent.

Reply:
boo but OK. In the future, we don't let consistency hold us back :)